src/libraries/System.Formats.Asn1/src/System/Formats/Asn1/AsnWriter.cs

```diff
@@ -730,7 +730,9 @@ internal Scope(AsnWriter writer)
 
             public void Dispose()
             {
-                if (_writer == null || _writer._nestingStack.Count == 0)
+                Debug.Assert(_writer == null || _writer._nestingStack != null);
+
+                if (_writer == null || _writer._nestingStack!.Count == 0)
```

Comment:
In case you're curious: The netcoreapp build complained that _nestingStack might be null, when neither the net461 or netstandard2.0 versions did (the field is declared as `Stack<StackFrame>?`, so I don't understand the difference in opinion).

The compound assert didn't make it happy, so it also got a ! suppression.

Reply:
The difference is that `netstandard2.0` and `net461` is not annotated so `Debug.Assert` doesn't make a difference.

Reply:
It is weird that you're getting a warning though because we have this:

https://github.com/dotnet/runtime/blob/9f485a58a1e80ab19e93b4b543329016eb9bcd5c/src/libraries/Directory.Build.targets#L6

Did you get it on VS or on command line?

Reply:
Command line, but when building the NetCoreAppCurrent flavor.  It makes sense, given the condition, since the two flavors it was already building are both suppressed.

Reply:
Ah I see. It is because the `Debug.Assert` is an ||.